modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/filter/provider/FileAspectRatioContentDashboardItemFilterProvider.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.content.dashboard.document.library.internal.item.filter.provider;
+
+import com.liferay.content.dashboard.document.library.internal.item.filter.FileAspectRatioContentDashboardItemFilter;
+import com.liferay.content.dashboard.item.action.exception.ContentDashboardItemActionException;
+import com.liferay.content.dashboard.item.filter.ContentDashboardItemFilter;
+import com.liferay.content.dashboard.item.filter.provider.ContentDashboardItemFilterProvider;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.util.Portal;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Mikel Lorza
+ */
+@Component(
+	property = "service.ranking:Integer=30",
+	service = ContentDashboardItemFilterProvider.class
+)
+public class FileAspectRatioContentDashboardItemFilterProvider
+	implements ContentDashboardItemFilterProvider {
+
+	@Override
+	public ContentDashboardItemFilter getContentDashboardItemFilter(
+			HttpServletRequest httpServletRequest)
+		throws ContentDashboardItemActionException {
+
+		return new FileAspectRatioContentDashboardItemFilter(
+			httpServletRequest, _language, _portal);
+	}
+
+	@Override
+	public String getKey() {
+		return null;
+	}
+
+	@Override
+	public ContentDashboardItemFilter.Type getType() {
+		return ContentDashboardItemFilter.Type.SUBMENU;
+	}
+
+	@Override
+	public boolean isShow(HttpServletRequest httpServletRequest) {
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-30087")) {
+			return false;
+		}
```

Comment:
Just double checking: is it ok for this to be a system ff?

Reply:
This should be an instance setting FF as you mentioned @adolfopa

Reply:
Yes, you are right, my mistake there.

thanks @adolfopa @jkappler